### PR TITLE
Update German Translation

### PR DESCRIPTION
**Pull request by heckelson**
**Resolved / Related Issues**

none

**Details of Changes**

I reviewed the German translation file (`Files/MultilingualResources/Files.de-DE.xlf`).

* looked for all occurences of `state="needs-review-translation"` in the file  and reviewed them, editing if necessary
* deleted all `<note>` entries, when no longer relevant
* Added translations to all occurences of `state="new"`

**Validation**
How did you test these changes?
- ~Built and ran the app~
- ~Tested the changes for accessibility~

### CODE REVIEW

**Comment by d2dyno1** [comment]
@R3voA3 fyi

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 1]
```diff
@@ -3697,43 +3644,43 @@ Wir benutzen das App Center um einen Überblick über die Nutzung von Files zu e
...
-          <target state="new">This setting modifies system files and can have unexpected side effects on your device. The developers take no responsibility in the event an issue occurs as a result. Continuing with this option is an acknowledgment of the risks involved with this action. Please note, uninstalling Files will not undo these changes and may prevent you from opening file explorer unless you turn off this setting before removing Files from your device.</target>
+          <target state="new">Diese Einstellung modifiziert Systemdateien und kann unerwartete Seiteneffekte mit sich bringen. Die Entwickler übernehmen keine Verantwortung, falls ein Problem auftritt. Mit dieser Einstellung fortzufahren ist eine Bestätigung dafür, dass Ihnen dieses Risiko bekannt ist. Bitte beachten Sie, dass das Deinstallieren von Files diese Änderungen nicht rückgängig macht. Es kann vorkommen, dass Sie den Explorer nicht mehr öffnen können, außer Sie deaktivieren diese Einstellung vor der Deinstallation von Files.</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.de-DE.xlf:3679]: Can you update the state to translated?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.de-DE.xlf (hunk 1 above)
Comment (heckelson) [reply Files/MultilingualResources/Files.de-DE.xlf]: Done! I guess I must have overlooked them, sorry!

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**